README.rst

```diff
@@ -1130,6 +1130,16 @@ the Chrome extension Don't Fuck With Paste.
 
 |MetLife|
 
+`Microsoft (e company store) <https://store.ecompanystore.com/microsoftevents/Shop/Register/PASSHOLDER#/>`__
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Max of 16 character oh and please don't use any characters we don'y know how to escape properly 
+also if it starts with ? you may break our shitty website. What out with your password generator 
```

Comment:
Can you try to keep this clean please?

Reply:
Happy to make the change just didn't think it mattered given the example directly above

Reply:
Thanks, I may indeed have missed some.